routers/slack_api.py

```diff
@@ -140,12 +157,12 @@ def slack_connect_redirect(request: Request):
             BotIntegration.objects.filter(pk=bi.pk).update(**config)
             bi.refresh_from_db()
 
+    redirect = connect(bi, current_run, published_run)
```

Comment:
This broke the creation of personal channels as it always set slack_create_personal_channels=False

Reply:
I believe sean requested the creation of personal channels disabled

Reply:
![image](https://github.com/user-attachments/assets/4e64cc8d-f4d5-4975-9be0-a83ce56c47ad)

Reply:
I did go through and test all the integrations. Would be good to have some sort of automated test for this too, even if it mocks the 3rd party portions

Reply:
Tracking here: https://app.asana.com/0/1203067047205953/1204911106314617

Reply:
I am not sure how we enable creation of personal channels in this case. There would be no way for end users to do it, so maybe we need some ux around this too